sdk/cosmos/azure-cosmos/setup.py

```diff
@@ -76,6 +76,6 @@
     install_requires=[
         "azure-core>=1.30.0",
         "typing-extensions>=4.6.0",
-        "aiohttp>=3.10.11"
+        "aiohttp>=3.0"
```

Comment:
We're removing this?

Reply:
We added this as part of the last hotfix. Now we only take the dependency on `ClientConnectionError` 
 which has been since aiohttp3.0 
https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientConnectionError